src/tespy/networks/networks.py

```diff
@@ -2606,6 +2734,78 @@ def print_components(c, *args):
         else:
             return np.nan
 
+    def print_exergy_conns(self):
+        r"""Print the calculation results of the (specific) physical exergy of
+        the connections to prompt.
+        """
+        df = pd.DataFrame(columns=['e_PH / (kJ / kg)', 'E_PH / MW'])
+        for c in self.conns.index:
+            row = (c.source.label + ':' + c.source_id + ' -> ' +
+                   c.target.label + ':' + c.target_id)
+            row_data = [c.ex_physical/10**3, c.Ex_Physical/10**6]
+            df.loc[row] = row_data
```

Comment:
Flexible unit system would be nice. The user will provide the unit and the method adapts correspondingly. I'll set this up.

Reply:
Thanks!